pkg/chartmuseum/server/multitenant/api.go

```diff
@@ -95,7 +95,7 @@ func (server *MultiTenantServer) deleteChartVersion(log cm_logger.LoggingFn, rep
 	return nil
 }
 
-func (server *MultiTenantServer) uploadChartPackage(log cm_logger.LoggingFn, repo string, content []byte, force bool) *HTTPError {
+func (server *MultiTenantServer) uploadChartPackage(log cm_logger.LoggingFn, repo string, content []byte, force bool) (string, *HTTPError ){
```

Comment:
Here will cause some compile problems since you not deal with the `error` branches ?

I think it's better to re-compile this PR after you rebased master ?

Reply:
Oh,my fault. It was caused by my careless when I resolved conflict. I have fixed it. Thanks for your review again, and sorry for the problems!